application/dash_application/views/preqc_wgs.py

```diff
@@ -204,10 +201,18 @@ def generate_mean_insert_size(df, colour_by, shape_by):
         "Base Pairs",
         colour_by,
         shape_by,
-        "none"
+        "none",
+        cutoff
     )
 
 
+def generate_reads_per_start_point(df, colour_by, shape_by, cutoff):
```

Comment:
Reads per start point wasn't one of the [graphs requested for pre-WGS](https://docs.google.com/spreadsheets/d/1HOozzs1jeQI3O4BiotMEGbSWoZnLSTCVEVjdtMZcFXM/edit#gid=0), does it need to be added?

Reply:
It's in the ticket though not in the spread sheet. :man_shrugging:

Reply:
Ah, sorry about that. I'll update the ticket.